basics/Operators/index.md

```diff
@@ -0,0 +1,55 @@
+# Java Operatos
+ Operators are used to perform operations on variables and values.
+
+## Types of Operators
+- Arithmetic operators
+- Assignment operators
+- Comparison operators
+- Logical operators
+- Bitwise operators
+
+### Arithmetic Operators
+Arithmetic operators are used to perform common mathematical operations.
+
+| Operator | Name         | Description                         | Example  |
+|----------|--------------|-------------------------------------|----------|
+| +        | Addition     | Adds together two values            | x + y    |
+| -        | Subtraction  | Subtracts one value from another    | x - y    |
+| *        | Multiplication| Multiplies two values               | x * y    |
+| /        | Division     | Divides one value by another        | x / y    |
+| %        | Modulus      | Returns the division remainder      | x % y    |
+| ++       | Increment    | Increases the value of a variable by 1 | ++x     |
+| --       | Decrement    | Decreases the value of a variable by 1 | --x     |
+
+### Assignment Operators
+Assignment operators are used to assign values to variables.
+| Operator | Example  | Same As       |
+|----------|----------|---------------|
+| =        | x = 5    | x = 5         |
+| +=       | x += 3   | x = x + 3     |
+| -=       | x -= 3   | x = x - 3     |
+| *=       | x *= 3   | x = x * 3     |
+| |=       | x /= 3   | x = x / 3     |
+| %=       | x %= 3   | x = x % 3     |
+| &=       | x &= 3   | x = x & 3     |
+| \|=       | x \|= 3   | x = x \| 3     |
+| ^=       | x ^= 3   | x = x ^ 3     |
+| >>=      | x >>= 3  | x = x >> 3    |
+| <<=      | x <<= 3  | x = x << 3    |
+### Comparison Operators
+Comparison operators are used to compare two values.
+| Operator | Name                         | Example  |
+|----------|------------------------------|----------|
+| ==       | Equal to                     | x == y   |
+| !=       | Not equal                    | x != y   |
+| >        | Greater than                 | x > y    |
+| <        | Less than                    | x < y    |
+| >=       | Greater than or equal to     | x >= y   |
+| <=       | Less than or equal to        | x <= y   |
+### Logical Operators
+Logical operators are used to determine the logic between variables or values.
+| Operator | Name         | Description                                   | Example                     |
```

Comment:
there is inconsistent around the table.

keep below column same everywhere.

Name
Operator
Example
Alternative
Description